static/admin/config.yml

```
@@ -1992,7 +1992,409 @@ collections:
+                  },
+                ],
+            }
+          - {
+              label: "Programs and Events Card",
+              name: "programsAndEvents",
+              widget: "list",
+              allow_add: true,
+              fields:
+                [
+                  {
+                    label: "Image",
+                    name: "image",
+                    widget: file,
+                    media_library: { config: { multiple: false } },
+                  },
+                  { label: "Title", name: "title", widget: "string" },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "string",
+                  },
+                  { label: "URL", name: "url", widget: "string" },
+                ],
+            }
+          - {
+              label: "Projects and Tools Section",
+              name: "projectsAndToolsInfo",
+              widget: "object",
+              collapsed: true,
+              fields:
+                [
+                  {
+                    label: "Row Reverse",
+                    name: "rowReverse",
+                    widget: boolean,
+                    required: false,
+                    default: false,
+                  },
+                  {
+                    label: "Pre Title",
+                    name: "preTitle",
+                    widget: "string",
+                    minimal: true,
+                  },
+                  {
+                    label: "Title",
+                    name: "title",
+                    widget: "string",
+                    minimal: true,
+                  },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "string",
+                    required: false,
+                  },
+                ],
+            }
+          - {
+              label: "Projects and Tools Card",
+              name: "projectsAndTools",
+              widget: "list",
+              allow_add: true,
+              fields:
+                [
+                  {
+                    label: "Image",
+                    name: "image",
+                    widget: file,
+                    media_library: { config: { multiple: false } },
+                  },
+                  { label: "Title", name: "title", widget: "string" },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "string",
+                  },
+                  {
+                    label: "Categories",
+                    name: "categories",
+                    widget: "list",
+                    allow_add: true,
+                    required: true,
+                    fields: [{ label: Label, name: label, widget: string }],
+                  },
+                  { label: "Date", name: "date", widget: "datetime" },
```

Maybe we could change this widget type to just `date`? Right now in the admin panel, when adding new project and tools card we have to pick a date and also a time. I don't think time is necessary here and we still be able to sort it just by date. What do you think?

```suggestion
                  { label: "Date", name: "date", widget: "date" },
```

---

Addressed in https://github.com/threshold-network/website/pull/94/commits/27449d584826e63585d71acade9675e371cc7d59